### ISSUE TITLE
Discounts do not reflect total sent to Xero

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/edd-xero#25]
If a customer uses a discount code, it does not appear to be using the discounted total when sending the amount to Xero - which throws off numbers in Xero. 

Customer Ticket:
https://secure.helpscout.net/conversation/436239167/67174/?folderId=749231

**Comment by mintplugins** [comment]
Another Customer Ticket:
https://secure.helpscout.net/conversation/449618265/68564?folderId=180499

**Comment by pippinsplugins** [comment]
@mintplugins ready for testing!

**Comment by pippinsplugins** [comment]
Nvm, not quite working.

**Comment by pippinsplugins** [comment]
Ok working now and ready for testing.

**Comment by mintplugins** [comment]
Replicated on `master`. Confirmed that the total sent to Xero is the correct total paid including the discount on `issue/25`. 

However, I noticed that there is a "discount" column in Xero that is blank. This was a $5.00 product with a 10% discount applied, which works out to $4.50. 

<img width="1237" alt="screen shot 2017-11-24 at 3 54 42 pm" src="https://user-images.githubusercontent.com/1638128/33223693-f88410ec-d12f-11e7-96ff-c00f3f83ca62.png">

I'm not sure if it's worth making that show the discounted percentage or not (probably not). This does make the numbers add up correctly in XERO so in my opinion this does fix the issue as reported.
